qutebrowser/keyinput/modeparsers.py

```diff
@@ -70,7 +71,7 @@ def _handle_single_key(self, e):
             self._debug_log("Ignoring key '{}', because the normal mode is "
                 "currently inhibited.".format(txt))
             return self.Match.none
-        if not self._keystring and any(txt == c for c in STARTCHARS):
+        if not self._keystring:
             message.set_cmd_text(self._win_id, txt)
             return self.Match.definitive
```

Comment:
This now causes qutebrowser to handle *any* keypress as an initial command char, which makes it crash when you press any key - you can get rid of this whole block.